src/tests.rs

```diff
@@ -405,7 +386,7 @@ fn test_bundle_parse_error() {
 // Test loading a bundle from a file
 #[test]
 fn test_bundle_deserialize() {
-    let bun = Bundle::from_file("testdata/bundle.json").unwrap();
+    let bun = Bundle::from_file("testdata/bundle.json").expect("parse testdata/bundle.json");
 
     assert_that(&bun.name).is_equal_to("helloworld".to_string());
     assert_that(&bun.schema_version).is_equal_to("v1.0.0-WD".to_string());
```

Comment:
Should we also update the test bundle to v1.0.0?

Reply:
Good catch! Yes we should.